recipes-kernel/linux/linux-96boards_4.4.bb

```diff
@@ -46,8 +46,8 @@ do_unpack_append_hikey() {
 
 do_unpack_mali_drv() {
     mkdir -p ${S}/drivers/gpu/arm
-    mv ${WORKDIR}/DX910-SW-99002-r6p0-01rel0/driver/src/devicedrv/mali \
-       ${S}/drivers/gpu/arm/utgard
+    mv ${WORKDIR}/DX910-SW-99002-r7p0-00rel1/driver/src/devicedrv/mali \
+           ${S}/drivers/gpu/arm/utgard
```

Comment:
unnecessary indentation changes

Reply:
will fix

Reply:
I can't seem to reply directly to your comment about X11 for some reason..so I'll do it here..no I haven't tested X11